Creat README.md for the application (#3)
caotuannghia20/recsys-taipy-app: README.md

> # How can this application do:

Reviewer: ```suggestion
# Movie Recommendation system application using Taipy
```

> - In the application, the user's information is secured as user_id

Reviewer: ```suggestion
- In the application, the user's information is secured as `user_id`.
```
Variable name should be put inside \` \`

- The app recommends **top_k** movies for a user via user_id, also give the **top_k actual** movies that the user is interested in.
- There are 2 models used to make recommendations: *kNN* and *MF*
- There are two metrics to evaluate the performance of the model: *recall* and *precision*. These are calculated automatically after making recommendations

> # How to run the application on a local machine

Reviewer: ```suggestion
## Run the application on a local machine
```

- Step 1: Clone this repo to the computer.
At the conmmand prompt, type <br/>
<code>git clone https://github.com/caotuannghia20/recsys-taipy-app.git </code>

>  - Step 2: Download the dataset [here](https://drive.google.com/file/d/1E_6jSGexFkjQmECkXEoegsp1KzoeOS4C/view?usp=sharing)

Reviewer: The link should lead to GroupLens page. Then there should be a section to describe how to split the data to train and test the recommendation model.

 - Step 3: Edit the dataset path in **config/config.py**
 - Step 4: Install the requirements enviroment, type : <br/>
 <code>pip install -r requirements.txt </code>
 - Step 5: At the conmmand prompt, type : <br/>
 <code>python main.py</code> <br/> to run the application on a local machine

> # How to use the application

Reviewer: ```suggestion
## Use the application
```

After run the application:

 We choose the model to make recommendations

> ![Choose the model to make recommendations](image/demo1.png )

Reviewer: Typo in the image.
**REOMMENDATION**

# Movie Recommendation system application using Taipy
- In the application, the user's information is secured as `user_id`.
- The app recommends **top_k** movies for a user via user_id, also give the **top_k actual** movies that the user is interested in.
- There are 2 models used to make recommendations: *kNN* and *MF*
- There are two metrics to evaluate the performance of the model: *recall* and *precision*. These are calculated automatically after making recommendations

## Run the application on a local machine

> - Step 1: Clone this repo to the computer.

Reviewer: ```suggestion
- Step 1: Clone the source code for Taipy application.
```

> At the conmmand prompt, type <br/>

Reviewer: ```suggestion
On your command prompt, run <br/>
```

<code>git clone https://github.com/caotuannghia20/recsys-taipy-app.git </code>

>  - Step 2: Download the dataset [here](https://grouplens.org/datasets/movielens/100k/)

Reviewer: ```suggestion
 - Step 2: Download the MovieLens 100K dataset [here](https://grouplens.org/datasets/movielens/100k/).
```

>  - Step 3: Split file `u.data` into training (**rating_train.csv**) and test (**rating_test.csv**) data with the ratio 80%/20% using file `ultis/splitdata.py`. Edit the path in line 17

Reviewer: ````suggestion
 - Step 3: Split file `u.data` into training (**rating_train.csv**) and testing (**rating_test.csv**) data.
First, edit the `TEST_RATIO` and `MOVIELENS_DATA_PATH` on `ultis/splitdata.py` then run:
```
python utils/splitdata.py
```
````
- [ ] Rename "ultis" to "utils" folder, short for **"utilities"**.

>  - Step 4: Edit the dataset path in `config/config.py` line 26, 34, 41

Reviewer: ```suggestion
 - Step 4: Edit the dataset path in `config/config.py` line 26, 34, 41
```
The path should be a constant, or at least a variable at the start of the file so that the user can easily find and edit.
